R/add_weights_cluster.R

```diff
@@ -0,0 +1,190 @@
+#' Add a column of cluster survey weights to a data frame.
+#'
+#' For use in surveys where you took a sample population out of a larger
+#' source population, with a cluster survey design.
+#'
+#' Will multiply the inverse chances of a cluster being selected, a household
+#' being selected within a cluster, and an individual being selected within a
+#' household.
+#' 
+#' As follows:
+#'
+#' ```
+#' (unique(cluster_cz) / unique(cluster_x)) *
```

Comment:
it took me a bit to figure out what was going on here until I realized you meant "number of unique clusters". It would probably be better to write it out in human language as opposed to a half-breed R language:

```
((clusters available) / (clusters surveyed)) *
((households in each cluster) / (households surveyed in each cluster)) *
((individuals eligible in each household) / individuals interviewed))
```

One thing I don't particularly understand is why the first two terms are inverse but the third term is not. Is there a reason for that?

Reply:
makes sense will change. 

The third term is also inverse! It is the equivalent of household source population / household sampled population